cmd/influx/export.go

```diff
@@ -0,0 +1,379 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/influxdata/influx-cli/v2/clients/export"
+	"github.com/influxdata/influx-cli/v2/pkg/cli/middleware"
+	"github.com/mattn/go-isatty"
+	"github.com/urfave/cli/v2"
+)
+
+func newExportCmd() *cli.Command {
+	var params struct {
+		out            string
+		stackId        string
+		resourceType   ResourceType
+		bucketIds      string
+		bucketNames    string
+		checkIds       string
+		checkNames     string
+		dashboardIds   string
+		dashboardNames string
+		endpointIds    string
+		endpointNames  string
+		labelIds       string
+		labelNames     string
+		ruleIds        string
+		ruleNames      string
+		taskIds        string
+		taskNames      string
+		telegrafIds    string
+		telegrafNames  string
+		variableIds    string
+		variableNames  string
+	}
+	return &cli.Command{
+		Name:  "export",
+		Usage: "Export existing resources as a template",
+		Description: `The export command provides a mechanism to export existing resources to a
+template. Each template resource kind is supported via flags.
+
+Examples:
+	# export buckets by ID
+	influx export --buckets=$ID1,$ID2,$ID3
+
+	# export buckets, labels, and dashboards by ID
+	influx export \
+		--buckets=$BID1,$BID2,$BID3 \
+		--labels=$LID1,$LID2,$LID3 \
+		--dashboards=$DID1,$DID2,$DID3
+
+	# export all resources for a stack
+	influx export --stack-id $STACK_ID
+
+	# export a stack with resources not associated with the stack
+	influx export --stack-id $STACK_ID --buckets $BUCKET_ID
+
+All of the resources are supported via the examples provided above. Provide the
+resource flag and then provide the IDs.
+
+For information about exporting InfluxDB templates, see
+https://docs.influxdata.com/influxdb/latest/reference/cli/influx/export/`,
+		Flags: append(
+			commonFlagsNoPrint(),
+			&cli.StringFlag{
```

Comment:
minor thing, but these could be sorted in lex order to make them easier to find

Reply:
Opened #133 to address this in general (it's a problem across commands).